docs/tutorials/simulations/MMsbm.ipynb

```diff
@@ -0,0 +1,170 @@
+    "\\vec{\\pi}_q &= \\begin{bmatrix} 0.25 & 0.25 & 0.25 & 0.25 \\end{bmatrix}\\\\\n",
+    "\\end{align*}\n",
+    "\n",
+    "According to $\\vec{\\pi}_p$, node $p$ will always behave as pertaining to the first community when interacting with other nodes, while node $q$ will interact 25% of the time as if belonging to community 1, 25% of the time as if belonging to community 2, etc.\n",
+    "\n",
+    "In MMSBM, this memership assignment is performed by sampling each membership vector $\\vec{\\pi}_p$ from a Dirichlet distribution with concentration parameter $\\alpha$ so that by modifying $\\alpha$ we can control the shape of the Dirichlet distribution and thus inderectly the assigned membership vectors. For example, when one of the entries in alpha is much greater than the others, the majority of the nodes will pertain to the community associated with that entry. If all entries of $\\alpha$ are identical to each other then values of the concentration parameter closer to $0$ will cause most of the nodes to belong in only one community similarly to an SBM. Always maintaining all entries identical, as we increse the values in $\\alpha$ more nodes are allowed to pertain to multiple communities, therefore the underlying block structure is progressively lost.\n",
+    "\n",
+    "To show this we sample a four-block SBM(undirected, no self-loops) graph with following parameters:\n",
+    "\\begin{align*}\n",
+    "k &= 4\\\\\n",
+    "n &= \\begin{bmatrix} 50 & 50 & 50 & 50 \\end{bmatrix}\\\\\n",
+    "B &=\\begin{bmatrix} 0.8 & 0.2 & 0.2 & 0.2 \\\\ 0.2 & 0.8 & 0.2 & 0.2 \\\\ 0.2 & 0.2 & 0.8 & 0.2 \\\\ 0.2 & 0.2 & 0.2 & 0.8 \\end{bmatrix}\n",
+    "\\end{align*}\n",
+    "\n",
+    "Similarly, we also sample from MMSBM a graph with the same number of communities, same connectivity matrix and same total number of vertices:\n",
+    "\\begin{align*}\n",
+    "n &=200\\\\\n",
+    "k &= 4\\\\\n",
+    "\\alpha &= \\begin{bmatrix} 0 & 0 & 0 & 0 \\end{bmatrix}\\\\\n",
+    "B &=\\begin{bmatrix} 0.8 & 0.2 & 0.2 & 0.2 \\\\ 0.2 & 0.8 & 0.2 & 0.2 \\\\ 0.2 & 0.2 & 0.8 & 0.2 \\\\ 0.2 & 0.2 & 0.2 & 0.8 \\end{bmatrix}\n",
+    "\\end{align*}\n",
+    "\n",
+    "Note that, differently from SBM, for the MMSBM we do not specify the number of vertices pertaining to each specific block as the assignment is performed via random sampling from Dirichlet distribution.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from graspologic.simulations import sbm, mmsbm\n",
+    "seed = 1\n",
+    "\n",
+    "rng = np.random.default_rng(seed)\n",
+    "np.random.seed(seed)\n",
+    "\n",
+    "n = [50] * 4\n",
+    "p = [[0.8,0.2,0.2,0.2],\n",
+    "    [ 0.2,0.8,0.2,0.2],\n",
+    "    [ 0.2,0.2,0.8,0.2],\n",
+    "    [ 0.2,0.2,0.2,0.8]]\n",
+    "\n",
+    "G_sbm = sbm(n, p, directed=False, loops=False)\n",
+    "\n",
+    "n= 200\n",
+    "k = 4\n",
+    "alpha = np.zeros(4)\n",
```

Comment:
I see that in your code, 0s in alpha were converted to very small values, but since alpha is defined as the Dirichlet parameter, I don't know if that would confuse people?

Reply:
The main reason why I initially added the approximations of zero to small values was because I thought it could make the tutorial and using the function more clear, although now that you brought this up, I agree that it may actually confuse people even more. I will change this both in the function and in the tutorial to avoid these confusions.